sdk/__tests__/port/out/FireblocksTransactionAdapter.test.ts

```diff
@@ -44,15 +45,13 @@ import * as path from 'path';
 
 const decimals = 6;
 const initialSupply = 1000;
-const maxSupply = 1000000;
 const apiSecretKey = fs.readFileSync(
 	path.resolve(FIREBLOCKS_SETTINGS.apiSecretKeyPath),
 	'utf8',
 );
 
 describe('🧪 FireblocksTransactionAdapter test', () => {
 	let stableCoinHTS: StableCoinViewModel;
-
 	const delay = async (seconds = 5): Promise<void> => {
```

Comment:
Never used. If it's a commonly used function I suggest to externalize it in a __tests__/utils.ts or something?